### PR TITLE
Feature/40835

### PR DESCRIPTION
**Pull request by UrvashiChaudhary**
1) Create a Trainee Component similar to other page component.
2) Inside Trainee Component create a folder named as components and create a new component AddDialog component
https://gyazo.com/3268e2b58ed78bd591db1ef3ce476c1a
3) AddDialog compoent will accept following props:
* open - boolean, will open the dialog if true.
* onClose - handler to close the dialog, passed from TraineeComponent
* onSubmit - handler to accept the dialog state for user: name, email, password
4) AddDialog will show information as show in the GIF
5) Add all the validations as shown in the GIF, similar to InputDemo component.
6) All the components used in the AddDialog must be from the Material UI.
7) TraineeComponent - Will hold the state of the Dialog, and update the state as per handler Request.
8) TraineeComponent - Will contain the button, and clicking on the button will open the dialog inside “AddDialog” component.

### CODE REVIEW

File: training/src/pages/Trainee/Trainee.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  handleClickOpen = () => {
+    this.setState({ open: true });
+  };
+
+  handleClose = () => {
+    const { open } = this.state;
+    this.setState({ open: false });
+    return open;
```
Comment (adilsuccessive) [inline training/src/pages/Trainee/Trainee.jsx]: return not required.

File: training/src/pages/Trainee/Trainee.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  };
+
+  handleSubmit = (data) => {
```
Comment (adilsuccessive) [inline training/src/pages/Trainee/Trainee.jsx]: on submit the Dialog does not closed.
also handleSubmit is not working properly.

**Review by adilsuccessive — CHANGES_REQUESTED** [review]

File: training/src/pages/Trainee/Trainee.jsx (hunk 1 above)
Comment (UrvashiChaudhary) [reply training/src/pages/Trainee/Trainee.jsx]: removed

File: training/src/pages/Trainee/Trainee.jsx (hunk 2 above)
Comment (UrvashiChaudhary) [reply training/src/pages/Trainee/Trainee.jsx]: resolved

**Review by adilsuccessive — APPROVED** [review]

**Pull request merged**